app/services/spree/orders/commissions_generator.rb

```diff
@@ -0,0 +1,20 @@
+module Spree
```

Comment:
@pmilewski this service should be aligned with standard Spree services eg. https://github.com/spree/spree/blob/master/core/app/services/spree/cart/set_quantity.rb